src/ConnectedMode/Migration/FixedStepsProgressToMigrationProgressConverter.cs

```diff
@@ -0,0 +1,40 @@
+﻿/*
+ * SonarLint for Visual Studio
+ * Copyright (C) 2016-2023 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+using System;
+using SonarLint.VisualStudio.ConnectedMode.Binding;
+
+namespace SonarLint.VisualStudio.ConnectedMode.Migration
+{
+    internal class FixedStepsProgressToMigrationProgressConverter : IProgress<FixedStepsProgress>
```

Comment:
I feel like the name is long enough to not write any documentation WDYT

Reply:
Yep, I don't think it needs a comment. However, I would change the suffix from `Converter` to `Adapter`, since it's an example of the [adapter pattern](https://en.wikipedia.org/wiki/Adapter_pattern). That would make the name even clearer.